test/common/http/codec_impl_corpus/response_204_B

```diff
@@ -0,0 +1,30 @@
+actions {
+  new_stream {
+    request_headers {
+      headers {
+        key: ":path"
+        value: "/"
+      }
+      headers {
+        key: ":method"
+        value: "GET"
+      }
+    }
+  }
+}
+actions {
+  client_drain {
+  }
+}
+actions {
+  stream_action {
+    response {
+      headers {
+        headers {
+          key: ":status"
+          value: "204"
```

Comment:
very minor nit, but can you add a response data action after this as a regression test? hopefully fuzzer works

Reply:
Done.